rafthttp/transport.go

```diff
@@ -180,7 +180,10 @@ func (t *Transport) Send(msgs []raftpb.Message) {
 		}
 		to := types.ID(m.To)
 
+		t.mu.RLock()
 		p, ok := t.peers[to]
+		t.mu.RUnlock()
```

Comment:
but when running, this cannot happen because we only AddRemote before sending messages.